source/_docs/configuration/entities_domains.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: "Entities and domains"
+description: "Describes what entities and domains are in Home Assistant."
+related:
+  - docs: /docs/configuration/state_object/
+    title: State object, entity state
+---
+
+Your {% term devices %} are represented in Home Assistant as entities. Entities are the basic building blocks to hold data in Home Assistant. An entity represents a {% term sensor %}, actor, or function in Home Assistant. Entities are used to monitor physical properties or to control other entities. An entity is usually part of a {% term device %} or a {% term service %}. Entities have [states](/docs/configuration/state_object/) and [attributes](#entity-attributes).
+
+All your entities are listed in the entities table, under {% my entities title="**Settings** > **Devices & services** > **Entities**" %}.
+
+<p class='img'><img src='/images/getting-started/entities.png' style='border: 0;box-shadow: none;' alt="Screenshot showing the Entities table">Screenshot of the Entities table. Each line represents an entity.</p>
+
+## Entity attributes
+
+Many entities have attributes. There are a few attributes that are used by Home Assistant for representing the entity in a specific way. Each integration will also have its own attributes to represent extra state data about the entity. For example, the light integration has attributes for the current brightness and color of the light. When an attribute is not available, Home Assistant will not write it to the state. The list of available attributes depends on the {% term device %}.
```

Comment:
Maybe give an example of these kinds of attributes:

> There are a few attributes that are used by Home Assistant for representing the entity in a specific way.

Reply:
I also think this sentence is hard to understand. I think we should rewrite it, but I'd like to get an example first and maybe that will clarify how to explain it.

Reply:
Good question. I'm not actually sure what this is referring to...I copied it from here: <https://www.home-assistant.io/docs/configuration/state_object/#attributes>. Maybe it's best to remove that sentence, on both pages?

Reply:
I think it's referring to the common state attributes that are available on all entities, eg `friendly_name` and `icon`.